code/DDSCodeTester.cpp

```diff
@@ -3969,372 +3969,518 @@ class CustomizedDataReaderListener : public DataReaderListener
-        //Change the period to 1 second
-        deadline.period.seconds = 1;
-        deadline.period.nanosec = 0;
+        // This example uses a DataWriter, but it can also be applied to: DataWriter, DataReader and Topic entities
+        DataWriterQos writer_qos;
+        // The DeadlineQosPolicy is default constructed with an infinite period.
+        // Change the period to 1 second
+        writer_qos.deadline().period.seconds = 1;
+        writer_qos.deadline().period.nanosec = 0;
+        // Use modified QoS in the creation of the corresponding entity
+        writer_ = publisher_->create_datawriter(topic_, writer_qos);
         //!--
     }
 
     {
         //DDS_CHANGE_DURABILITY_QOS_POLICY
-        DurabilityQosPolicy durability;
-        //The DurabilityQosPolicy is default constructed with kind = VOLATILE_DURABILITY_QOS
-        //Change the kind to TRANSIENT_LOCAL_DURABILITY_QOS
-        durability.kind = TRANSIENT_LOCAL_DURABILITY_QOS;
+        // This example uses a DataWriter, but it can also be applied to: DataWriter, DataReader and Topic entities
+        DataWriterQos writer_qos;
+        // The DurabilityQosPolicy is default constructed with kind = VOLATILE_DURABILITY_QOS
+        // Change the kind to TRANSIENT_LOCAL_DURABILITY_QOS
+        writer_qos.durability().kind = TRANSIENT_LOCAL_DURABILITY_QOS;
+        // Use modified QoS in the creation of the corresponding entity
+        writer_ = publisher_->create_datawriter(topic_, writer_qos);
         //!--
     }
 
     {
         //DDS_CHANGE_ENTITY_FACTORY_QOS_POLICY
-        EntityFactoryQosPolicy entity_factory;
-        //The EntityFactoryQosPolicy is default constructed with autoenable_created_entities = true
-        //Change it to false
-        entity_factory.autoenable_created_entities = false;
+        // This example uses a Participant, but it can also be applied to:
+        // DomainParticipantFactory, DomainParticipant, Publisher and Subscriber entities
+        DomainParticipantQos participant_qos;
+        // The EntityFactoryQosPolicy is default constructed with autoenable_created_entities = true
+        // Change it to false
+        participant_qos.entity_factory().autoenable_created_entities = false;
+        // Use modified QoS in the creation of the corresponding entity
+        participant_ = factory_->create_participant(domain, participant_qos);
         //!--
     }
 
     {
         //DDS_CHANGE_GROUP_DATA_QOS_POLICY
-        GroupDataQosPolicy group_data;
-        //The GroupDataQosPolicy is default constructed with an empty collection
-        //Collection is a private member so you need to use getters and setters to access
-        //Add data to the collection
-        std::vector<eprosima::fastrtps::rtps::octet> vec;
-        vec = group_data.data_vec(); // Getter function
+        // This example uses a Publisher, but it can also be applied to: Publisher and Subscriber entities
```

Comment:
The same as before, it sounds weird to me for you to say it can also be applied to a `Publisher`. I would remove it. Please check because the same issue takes place all over the code.